book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @author = author
+    @title = title
+    @rentals = []
+  end
+end
```

Comment:
- [ ] they should be a method for adding new rental the method should look like this :point_down: 
```
def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
 ....
  end

```